Engineering/BestPractices/AgileManifesto.md

```diff
@@ -0,0 +1,39 @@
+# Learn about Agile Manifesto and try to follow its values and principles
+
+## (https://agilemanifesto.org)
+
+### We are uncovering better ways of developing software by doing it and helping others do it. Through this work we have come to value:
+
+* Individuals and interactions over processes and tools
+
+* Working software over comprehensive documentation
+
+* Customer collaboration over contract negotiation
+
+* Responding to change over following a plan
+
+### Principles behind the Agile Manifesto:
```

Comment:
Why not just link to the principles at http://agilemanifesto.org/principles.html ?